installer/seldon-core/description.yaml

```diff
@@ -0,0 +1,23 @@
+name: seldon-core
+product: seldon-core
+version: "1.10.0"
+description: |
+  An MLOps framework to package, deploy, monitor and manage thousands of production machine learning models
+namespace: seldon-system
+install:
+  - type: helm
+    chart: seldon-core-operator
+    repo: https://storage.googleapis.com/seldon-charts
+    values: values.yaml
+    waitfor:
+      - kind: deployment
+        selector: control-plane=seldon-controller-manager
+        namespace: seldon-system
+        condition: available
+        timeout: 600
+uninstall:
+  - type: helm
+gateways:
+  - name: seldon
+    servicehost: seldon-core
+    port: 80
```

Comment:
don't forget to register the extension services, similar to how we do it for kfserving

Reply:
Yeah, I know, I'll add it once I am able to run the example

Reply:
the extension services are still missing